subworkflows/local/bed_consensus_quantify_qc_bedtools_featurecounts_deseq2.nf

```diff
@@ -90,8 +90,8 @@ workflow BED_CONSENSUS_QUANTIFY_QC_BEDTOOLS_FEATURECOUNTS_DESEQ2 {
         }
         .join(ch_bams)
         .map {
-            antibody, meta, saf, bams ->
-                [ meta, bams.flatten().sort(), saf ]
+            antibody, meta, saf, bams, meta_single_end ->
```

Comment:
the comment of `ch_bams` at line 15 has to be updated, it confused me about the content of this channel here

Reply:
Good spot!